pynestml/utils/ast_utils.py

```diff
@@ -859,3 +858,44 @@ def get_statements_from_block(cls, var_name, block):
                and node.small_stmt.get_assignment().lhs.get_name() == var_name:
                 stmts.append(node)
         return stmts
+
+    @classmethod
+    def has_delay_variable(cls, node: ASTFunctionCall) -> bool:
+        """
+        Checks if the given function call is actually a variable with a delay differential part
+        :param node: The expression
+        """
+        # Check if the function name is a state variable
```

Comment:
Could you re-write this function as a one-liner that just checks whether ``get_delay_variable_symbol(node)`` returns ``None``?